qutip/utilities.py

```diff
@@ -465,6 +471,8 @@ def _rmse(fun, xdata, ydata, params):
     parameters. (The closer to zero = the better the fit.)
     """
     yhat = _evaluate(fun, xdata, params)
+    if (np.max(ydata) - np.min(ydata)) == 0.0:
+        return 0
```

Comment:
What are you trying to do here? The rmse is not necessarily zero if the data is constant

Reply:
Just avoiding outputting rmse being nan.  It is the case for the fittings considered, but I can just change it to  

```
if (yhat == ydata).all(): 
    return 0
```
To keep _rmse fully general

Reply:
Aah no, I guess it is fine as it is, thank you for clarifying!